discojs/src/client/decentralized/decentralized_client.ts

```diff
@@ -22,9 +22,14 @@ export class DecentralizedClient extends Client {
   /**
    * The pool of peers to communicate with during the current training round.
    */
-  private pool?: PeerPool
-  private connections?: Map<NodeID, PeerConnection>
-  
+  #pool?: PeerPool
+  #connections?: Map<NodeID, PeerConnection>
+  #roundParticipants = 1
+
+  get nbOfParticipants(): number {
+    return this.#roundParticipants
```

Comment:
maybe we can avoid `#roundParticipants` entirely?

```suggestion
    return this.aggregator.nodes.size
```

Reply:
hmm I had to rely on another variable because the aggregator's nodes are reset after exchanging the weights (and forgot to document it) but thinking about it now I think we can simply stop resetting the nodes.